opentelekomcloud/acceptance/ecs/resource_opentelekomcloud_ecs_instance_v1_test.go

```diff
@@ -141,6 +141,7 @@ func TestAccEcsV1InstanceEncryption(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckEcsV1InstanceExists(resourceInstanceV1Name, &instance),
 					resource.TestCheckResourceAttr(resourceInstanceV1Name, "data_disks.0.kms_id", env.OS_KMS_ID),
+					resource.TestCheckResourceAttr(resourceInstanceV1Name, "system_disk_kms_id", env.OS_KMS_ID),
```

Comment:
Please add validation of `env.OS_KMS_ID` to the test, so it won't be marked as passing if it's not set.

Reply:
done